tools/build_includes/modules-with-dependencies-BE.yaml

```diff
@@ -1,73 +1,19 @@
 # Code generated by tools/build.go; DO NOT EDIT.
 - add: /ee/be/modules/040-node-manager/openapi/config-values.yaml
   to: /deckhouse/modules/040-node-manager/openapi/config-values.yaml
-  excludePaths:
```

Comment:
Why do we delete here?

Reply:
This is pointless for single files, they will end up in the image anyway. 
The excludePaths are added to the path above.